charts/dex/templates/service.yaml

```diff
@@ -32,6 +32,9 @@ spec:
 {{- end }}
 {{- if hasKey .Values.service "loadBalancerIP" }}
   loadBalancerIP: {{ .Values.service.loadBalancerIP }}
+{{- end }}
```

Comment:
Commit message talks about the actual Dex cluster IP being the 7th Service IP. This is true, but not related to this commit: explanation should be added to the "Deploy Dex" commit.